splice_spark2/src/main/spark2.4/com/splicemachine/spark2/splicemachine/KafkaToDF.scala

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
Is KafkaToDF platform dependent and has to duplicated for spark 2.2, 2.3, and 2.4?

Reply:
KafkaToDF passes data into Spark classes and they may be subject to change from one Spark version to another.  The Kafka classes being used may change between Kafka versions.  KafkaConsumer.poll(long) was recently deprecated in favor of KafkaConsumer.poll(Duration).